public/src/bc5SupportDev/java/com/revenuecat/purchases/models/PricingPhase.kt

```diff
@@ -0,0 +1,19 @@
+package com.revenuecat.purchases.models
+
+import android.os.Parcelable
+import com.android.billingclient.api.ProductDetails.RecurrenceMode
+import com.revenuecat.purchases.parceler.JSONObjectParceler
+import kotlinx.parcelize.Parcelize
+import kotlinx.parcelize.TypeParceler
+import org.json.JSONObject
+
+@Parcelize
+@TypeParceler<JSONObject, JSONObjectParceler>()
+data class PricingPhase(
+    val billingPeriod: String,
+    val priceCurrencyCode: String,
+    val formattedPrice: String,
+    val priceAmountMicros: Long,
+    @RecurrenceMode val recurrenceMode: Int, // todo should we make our own version of recurrencemode?
```

Comment:
Yes, I think so - if not we'll be leaking values from Google directly and we could run into problems somewhere down the line.

Reply:
It's a good opportunity to make it an enum

Reply:
I'm generally against using enums (for outputs of an API at the very least), cause they tend to cause ugly errors when new stuff is added.

If Google adds a new value as an int, right now we just ignore it cause it's not matching any values we know.
But trying to coerce a new value into an Enum that doesn't define it will throw an ugly exception and break the client.

Reply:
what we've done is provide an UNKNOWN value if the billingclient value is unexpected. but they can't add a new value without a new version of the billing client, so we should know